app/src/main/res/layout/profile_row_item.xml

```diff
@@ -5,87 +5,101 @@
     android:id="@+id/frameLayout2"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:padding="8dp">
+    android:paddingVertical="5dp">
```

Comment:
Should we at some point unify the padding throughout the app and store those in dimens ?